.github/workflows/test.yml

```diff
@@ -3,26 +3,44 @@ name: Build
 on:
   push:
     branches:
-      - master
+      - rewrite
   pull_request:
     branches:
-      - master
+      - rewrite
 
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version:
-          - '12'
-          - '14'
-          - '16'
+        node-version: ['14.x', '16.x']
+
     steps:
-     - uses: actions/checkout@v2
-     - uses: actions/setup-node@v2
-       with:
-         node-version: ${{ matrix.node-version }}
-     - name: Install dependencies
-       run: npm ci
-     - name: Run tests
-       run: npm t
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Cache modules
```

Comment:
Does this have an effect when using `npm ci`? Will `~/.npm` contain a global cache that is used on `npm ci`?

Reply:
It just speeds up installs dramatically, if no changes are in package-lock.json.